### PR TITLE
New workshop request

### PR DESCRIPTION
**Pull request by csmanea**
#WMS ID: 11368: "Ignite your Oracle Infrastructure Services - ADVANCED"

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by aberinnj** [comment]
Hey Cristian, a couple of your images do not match the naming conventions. To name a few:

- create-vcn/images/VCN-final.png
- iam/images/image020.png

**Comment by anooshapilli** [comment]
@csmanea - This workshop is in submitted status in WMS. Your workshop needs to be approved and should be in Self QA Complete status with the Self QA Checklist filled out to be able to review and approve the PR.

**Review by anooshapilli — COMMENTED** [review]
After the workshop is approved, please fill out the Self QA Checklist and update the workshop status to Self QA Complete to review the PR

**Comment by csmanea** [comment]
Hi @anooshapilli, thanks for the response.

I think there might be a bug, we finished Self-QA before PR. 
Checked again, and there's a check mark that won't stay checked, and therefore won't change the status to Self QA complete.

Please see the attached.
image (1)

**Comment by aberinnj** [comment]
@csmanea You may have needed to click on Save, nevertheless, it's now **selected**. 

Also I still see some files with uppercase characters, for example

- create-vcn/images/navigate to SL.png

**Comment by csmanea** [comment]
> @csmanea You may have needed to click on Save, nevertheless, it's now **selected**.
> 
> Also I still see some files with uppercase characters, for example
> 
> * create-vcn/images/navigate to SL.png

done, anything else?

File: oci-ignite-advanced/intro/intro.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### Objectives
+
+In this workshop, you will:
+
+- Get introduced to the regions available on Oracle Cloud
+- Apply the Ignite Basics knowledge
+- Create a Virtual Cloud Network
+- Create an Autonomous Database
+
+**<font color="blue">Optional</font>**:
```
Comment (anooshapilli) [inline oci-ignite-advanced/intro/intro.md:23]: We do not recommend having html code in markdown files and coloring of the text. Please remove the optional heading. You have that point part of Objectives and add optional in the braces.

File: oci-ignite-advanced/intro/intro.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ - Create a Kubernetes Cluster
+### Prerequisites
+* [Familiarity with OCI console](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/console.htm)
+* [Overview of Networking](https://docs.oracle.com/en-us/iaas/Content/Network/Concepts/overview.htm)
+* [Oracle Cloud basics](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/concepts.htm)
+* [Connecting to a compute instance](https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/accessinginstance.htm)
+* Your **<font color="red">Oracle Cloud Account</font>** - During this workshop, we will create a Level 100 environment for you to use on your tenancy.
```
Comment (anooshapilli) [inline oci-ignite-advanced/intro/intro.md:30]: Same here, please remove the html code.

File: oci-ignite-advanced/intro/intro.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+
+## About Oracle Cloud Infrastructure
+
+Oracle Cloud Infrastructure is a set of complementary cloud services that enable you to build and run a wide range of applications and services in a highly available hosted environment. Oracle Cloud Infrastructure offers high-performance compute capabilities (as physical hardware instances) and storage capacity in a flexible overlay virtual network that is securely accessible from your on-premises network.
+
+### Let's get Started!
```
Comment (anooshapilli) [inline oci-ignite-advanced/intro/intro.md:37]: Remove the Let's get started section and add You may now **proceed to the next lab**.

File: oci-ignite-advanced/intro/intro.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ - Create a Kubernetes Cluster
+### Prerequisites
+* [Familiarity with OCI console](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/console.htm)
+* [Overview of Networking](https://docs.oracle.com/en-us/iaas/Content/Network/Concepts/overview.htm)
+* [Oracle Cloud basics](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/concepts.htm)
+* [Connecting to a compute instance](https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/accessinginstance.htm)
+* Your **<font color="red">Oracle Cloud Account</font>** - During this workshop, we will create a Level 100 environment for you to use on your tenancy.
+
+
+## About Oracle Cloud Infrastructure
```
Comment (anooshapilli) [inline oci-ignite-advanced/intro/intro.md:33]: Move this section after ETA and before objectives

File: oci-ignite-advanced/workshops/tenancy/manifest.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            "title": "Introduction",
+            "filename": "./../../intro/intro.md"
+        },
+        {
+            "title": "Get Started",
+            "description": "Login to Oracle Cloud",
+            "filename": "https://oracle-livelabs.github.io/common/labs/cloud-login/cloud-login.md"
+        },
+        {
+            "title": "Ignite Basics: Identity and Access Management",
```
Comment (anooshapilli) [inline oci-ignite-advanced/workshops/tenancy/manifest.json:15]: Labs should start after the Get Started Lab. Please add Lab 1 in the title and update the Lab numbers in the rest of the lab titles in this file.

File: oci-ignite-advanced/iam/identity-access-management-identity-domain.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+- Create a user
+- Create a group
+- Create a policy associated to the group
+- Add user to the group
+
+### Prerequisites
+
+* This lab requires completion of the **Get Started** section in the Contents menu on the left.
+
+* Your **<font color="red">Oracle Cloud Account</font>** - During this workshop we will create a basic environment for you to use on your tenancy.
```
Comment (anooshapilli) [inline oci-ignite-advanced/iam/identity-access-management-identity-domain.md:26]: remove the html code

File: oci-ignite-advanced/iam/identity-access-management-identity-domain.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+
+## Task 1: Create a Compartment
+
+A compartment is a collection of cloud assets, like compute instances, load balancers, databases, etc. By default, a root compartment was created for you when you created your tenancy (i.e. when you registered for the trial account). It is possible to create everything in the root compartment, but Oracle recommends that you create sub-compartments to help manage your resources more efficiently.
+
+1. Click the **Navigation Menu** in the upper left, navigate to **Identity & Security**, and select **Compartments**.
+
+ ![Navigation Menu](images/id-compartments.png " ")
+
+1. Click **Create Compartment**.
```
Comment (anooshapilli) [inline oci-ignite-advanced/iam/identity-access-management-identity-domain.md:37]: Please fix the numbering in the tasks.

File: oci-ignite-advanced/iam/identity-access-management-identity-domain.md [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+
+A user's permission to access services comes from the _groups_ to which they belong. The permissions for a group are defined by policies. Policies define what actions members of a group can perform, and in which compartments. Users can access services and perform operations based on the policies set for the groups of which they are members.
+
+We'll create a user, a group, and a security policy to understand the concept.
+
+In 2022, OCI IAM introduced Identity Domains. An identity domain is a container for managing users and roles, federating and provisioning users, secure application integration through Oracle Single Sign-On (SSO) configuration, and OAuth administration.
+
+1. Click the **Navigation Menu** in the upper left. Navigate to **Identity & Security** and select **Domains**
+
+For IAM with Identity Domains, what was identified before as IAM users and groups, now is under the default domain.
```
Comment (anooshapilli) [inline oci-ignite-advanced/iam/identity-access-management-identity-domain.md:56]: Indent this text with 4 spaces from begin of the line.

File: oci-ignite-advanced/iam/identity-access-management-identity-domain.md [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+
+   ![Create Group](images/id-domains-create-group.png) 
+
+1. Click your new group to display it. Your new group is displayed.
+
+   ![New group is shown](images/id-domains-group-detail.png)
+
+1. Create a New User
+
+   a) In the breadcrumb, click on **Default Domain**
```
Comment (anooshapilli) [inline oci-ignite-advanced/iam/identity-access-management-identity-domain.md:84]: We recommend only using numbers followed by dot( . ) or underordered lists using dashes ( - ) or asterisks ( * ). Please remove a) b) c) format

File: oci-ignite-advanced/create-adw/autonomous-data-warehouse.md [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+Please *proceed to the next lab*.
+
+## Learn more
+
+See the [documentation](https://docs.oracle.com/en/cloud/paas/autonomous-data-warehouse-cloud/user/autonomous-workflow.html#GUID-5780368D-6D40-475C-8DEB-DBA14BA675C3) on the typical workflow for using Autonomous Data Warehouse.
+
+
+## Acknowledgements
+*Congratulations! You have successfully completed the lab.*
```
Comment (anooshapilli) [inline oci-ignite-advanced/create-adw/autonomous-data-warehouse.md:215]: Delete this line here or move it about the Learn more section

File: oci-ignite-advanced/create-oke/create-oke-cluster.md [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+    - __Name__ - Enter a memorable name for the database for display purposes. For this lab, use __Ignite-Cluster__.
+    - __Kubernetes version__ - Leave the default option.    
+
+    ![Enter the required details.](images/configure-cluster-oke1.png " ")
+
+6. Choose the **Public Endpoint** in the Kubernetes API endpoint to easily access the kubectl. *If you use a Bastion or an admin instance to access the kubectl commands, select the **Private Endpoint**.*
+
+  Select **Managed** for the **Node type**, and **Private workers** for the **Kubernetes worker nodes**.
+
+*Optionally choose the visibility of the Kubernetes Worker Nodes to Public Workers if you want public access to your nodes and the number of nodes to 2 or 1 if you want a reduced number of nodes*
```
Comment (anooshapilli) [inline oci-ignite-advanced/create-oke/create-oke-cluster.md:53]: indent this text

**Review by anooshapilli — COMMENTED** [review]
Please check all the labs with the review shared. Also, the folder name markdown file name needs to match, please update the lab markdown file name to match the folder name, for example, create-adw folder should have the filename create-adw.md. After the file name, please update the manifest.json file as well.

File: oci-ignite-advanced/workshops/tenancy/manifest.json (hunk 5 above)
Comment (csmanea) [reply oci-ignite-advanced/workshops/tenancy/manifest.json:15]: I will move our Ignite Basics section before the Get Started as it is not an actual lab, but a way to revise the knowledge from the first part of the workshop.

File: oci-ignite-advanced/create-oke/create-oke-cluster.md (hunk 11 above)
Comment (csmanea) [reply oci-ignite-advanced/create-oke/create-oke-cluster.md]: done.

File: oci-ignite-advanced/iam/identity-access-management-identity-domain.md (hunk 9 above)
Comment (csmanea) [reply oci-ignite-advanced/iam/identity-access-management-identity-domain.md]: done.

File: oci-ignite-advanced/create-adw/autonomous-data-warehouse.md (hunk 10 above)
Comment (csmanea) [reply oci-ignite-advanced/create-adw/autonomous-data-warehouse.md]: done.

File: oci-ignite-advanced/iam/identity-access-management-identity-domain.md (hunk 8 above)
Comment (csmanea) [reply oci-ignite-advanced/iam/identity-access-management-identity-domain.md]: done.

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**